### ISSUE TITLE
If a EZ-Flash Omega is inserted, TwiLightMenu takes more than a minute to load.

### ISSUE THREAD

**Issue by McFly75** [DS-Homebrew/TWiLightMenu#1937]
When I have a EZ-Flash Omega inserted in my DS Lite, it takes more than a minute to load TwiLightMenu v24.8.0.

When I do not have EZ-Flash Omega inserted it takes about 6-7 seconds.

I have tried several Slot-1 flash cards and they all take more than a minute to load TwilIghtMenu if a EZ-Flash Omega is inserted.
(CycloDS, Supercard DSOnei, DSTT)

And if I have a EZ-Flash IV or a EZ-Flash V/3in1 inserted, it only takes about 5-6 seconds to load TwiLightMenu
(so even faster than without a Slot-2 flash card inserted ?)

**Comment by McFly75** [comment]
I think that you are right. That the "rumble update" is the reason that it takes more than a minute to boot TwiLightMenu if a EZ-Flash Omega card is inserted.

I tested this by installing TwiLightMenu v24.7.0 (does NOT have the "rumble update") and TwiLightMenu boots up in about 6 seconds with the EZ-Flash Omega inserted (on a Gateway Blue slot-1 card with only TwiLightMenu installed)

**Comment by McFly75** [comment]
Found a (temporary) fix.
Switch Theme to Homebrew Launcher (no box art in this theme)

**Comment by RocketRobz** [comment]
Any other theme that isn't the DSi theme will work as well.

**Comment by McFly75** [comment]
Yes I just found that out. I really like the 3DS theme, that also has box art support.
